### ISSUE TITLE
Uncaught TypeError: Cannot read property 'endsWith' of null

### ISSUE THREAD

**Issue by kaelZ** [gangstead/atom-scala-format#4]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.174.0
**System**: linux 3.16.0-23-generic
**Thrown From**: [scala-format](https://github.com/gangstead/atom-scala-format) package, v0.1.2
### Stack Trace

Uncaught TypeError: Cannot read property 'endsWith' of null

```
At /home/kael/.atom/packages/scala-format/lib/scala-format.js:61

TypeError: Cannot read property 'endsWith' of null
  at Object.module.exports.format (/home/kael/.atom/packages/scala-format/lib/scala-format.js:61:17)
  at atom-workspace.<anonymous> (/home/kael/.atom/packages/scala-format/lib/scala-format.js:23:22)
  at atom-workspace.handler (/usr/share/atom/resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/usr/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/usr/share/atom/resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom/resources/app/src/command-registry.js:156:19)
  at Ipc.<anonymous> (/usr/share/atom/resources/app/src/window-event-handler.js:65:30)
  at Ipc.emit (events.js:107:17)
  at process.<anonymous> (/usr/share/atom/resources/atom/renderer/api/lib/ipc.js:22:29)
  at process.emit (events.js:110:17)

```
### Commands

```
     -0:00.0 scala-format:format (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
scala-format, v0.1.2

# Dev
No dev packages
```

/cc @atom/core

**Comment by gangstead** [comment]
Dupe of https://github.com/gangstead/atom-scala-format/issues/1 which I'm testing a fix for right now.

**Comment by gangstead** [comment]
Fixed in 0.1.3 with Atom 0.181 or above.
